package.json

```diff
@@ -72,6 +73,8 @@
     "@angular/compiler-cli": "18.2.5",
     "@angular/language-service": "18.2.5",
     "@chromatic-com/storybook": "2.0.2",
+    "@fortawesome/angular-fontawesome": "0.15.0",
+    "@fortawesome/free-solid-svg-icons": "6.6.0",
```

Comment:
the icons are from fortawesome using their latest angular package which is different in the original codebase.  The original codebase used css styles and <i> tags to accomplish the same thing.  This ends up being a little cleaner.  In order to support animations, I wasn't able to easily move away from @fortawesome.